packages/terra-button/src/_mixins.scss

```diff
@@ -75,3 +75,65 @@
     color: map-get($color-map, disabled-color);
   }
 }
+
+@mixin terra-consumer-theme-button($color) {
+  $color-dark-30: mix($color, #000, 70%);
```

Comment:
Should we pull $color-dard-30 through $color-light-64 out into their own mixin since they are reusable?